Rubberduck.Core/UI/About/AboutControlViewModel.cs

```diff
@@ -47,6 +49,29 @@ public CommandBase UriCommand
             }
         }
 
+        private CommandBase _viewLogCommand;
```

Comment:
Commands bound to WPF views probably shouldn't be lazily instantiated - it makes the `EvaluateCanExecute` bindings behave oddly.

Reply:
For my own projects, what I would typically do is have no private field, just:

    public AboutControlViewModel(IVersionCheck version)
    {
        _version = version;
        ViewLogCommand = new DelegateCommand(...);
    }
    
    ...
    
    public CommandBase ViewLogCommand { get; }

Is that more in the lines of what you were thinking? If so, should I do the same with `UriCommand`?

Reply:
That's the route I'd go (and yeah, if you want to catch the `UriCommand` too that would be great.  Given what the dialog actually _does_, this is mainly a nitpick but I've been trying to clean these up as I run across them.